teamsyncd/teamsyncd.cpp

```diff
@@ -34,7 +42,13 @@ int main(int argc, char **argv)
             while (true)
             {
                 Selectable *temps;
-                s.select(&temps);
+                s.select(&temps, 1000); // check every second
+
+                auto diff = duration_cast<seconds>(steady_clock::now() - start_time);
+                if(diff.count() > WR_PENDING_TIME)
```

Comment:
>WR_PENDING_TIME [](start = 34, length = 15)

If we dynamically change WR_PENDING_TIME to 0 in cold reboot, you may have unified logic in teamsync

Reply:
currently it the time is 0, we use default timer

Reply:
You code is good now. Actually I mean you can treat the cold reboot as a special warm reboot with a timer 0. So it just create view and apply view in 1 second.